### DISCUSSION TITLE
Unable view the published stream from a browser

### DISCUSSION THREAD

**Discussion by joeswaminathan** [bluenviron/mediamtx#3592] [Questions]
### Question

I am publishing a video stream using the following command
ffmpeg -re -stream_loop -1 -i sample_960x400_ocean_with_audio.ts -c copy -f rtsp rtsp://localhost:8554/mystream

The server log confirms it

2024/08/01 01:16:08 INF [RTSP] [conn 127.0.0.1:35558] opened
2024/08/01 01:16:08 INF [RTSP] [session a49af6e5] created by 127.0.0.1:35558
2024/08/01 01:16:08 INF [RTSP] [session a49af6e5] is publishing to path 'mystream', 2 tracks (MPEG-1/2 Video, MPEG-1/2 Audio)

But when I try to access the stream from a browser I get an error, and the server log shows

2024/08/01 01:17:01 INF [RTSP] [conn 3.130.77.35:53516] opened
2024/08/01 01:17:01 INF [RTSP] [conn 3.130.77.35:53516] closed: invalid URL (/mystream)
2024/08/01 01:17:01 INF [RTSP] [conn 3.130.77.35:63005] opened
2024/08/01 01:17:01 INF [RTSP] [conn 3.130.77.35:54882] opened
2024/08/01 01:17:02 INF [RTSP] [conn 3.130.77.35:63005] closed: invalid URL (/mystream)
2024/08/01 01:17:02 INF [RTSP] [conn 3.130.77.35:54882] closed: EOF
2024/08/01 01:17:03 INF [RTSP] [conn 3.130.77.35:18647] opened
2024/08/01 01:17:03 INF [RTSP] [conn 3.130.77.35:18647] closed: invalid URL (/mystream)
2024/08/01 01:17:03 INF [RTSP] [conn 3.130.77.35:12476] opened
2024/08/01 01:17:07 INF [RTSP] [conn 3.130.77.35:12476] closed: EOF
2024/08/01 01:17:08 INF [RTSP] [conn 3.130.77.35:9004] opened
2024/08/01 01:17:08 INF [RTSP] [conn 3.130.77.35:9004] closed: invalid URL (/mystream)
2024/08/01 01:17:08 INF [RTSP] [conn 3.130.77.35:4877] opened
2024/08/01 01:17:09 INF [RTSP] [conn 3.130.77.35:52096] opened
2024/08/01 01:17:12 INF [RTSP] [conn 3.130.77.35:4877] closed: EOF
2024/08/01 01:17:12 INF [RTSP] [conn 3.130.77.35:52096] closed: EOF

What am I doing wrong. I am using the default yml that is present on the github and didn't make any changes to it.

**Comment by aler9** [comment]
In order to read a stream with a browser, you have to reach the WebRTC server (port 8889), not the RTSP server (port 8554):

```
http://localhost:8889/mystream
```

[a comment by an automated account is not shown]
